### ISSUE TITLE
Server down, owner missing!

### ISSUE THREAD

**Issue by Amaz1** [CraigyDavi/Craig-Server_game#34]
Now the server has been down for nearly/over a month, I'm wondering if @CraigyDavi and the server are alright and if the server will be going up ever again... 
If you get this Craig, a reply would be greatly appreciated!

To anyone else:
![](http://i.imgur.com/yN3cICk.jpg)
Made by Gabo.

**Comment by CraigyDavi** [comment]
Hi Amaz,

Sorry that I've left everyone in the dark. I've been super busy at the moment and I felt this was having a negative effect on the server, this is why I took it down. For example I sometimes noticed how I was needed to sort out a whole variety of issues and I felt bad that I couldn't due to how busy I was. 

Not to worry though. I hope to start bringing the server back to life soon so expect the server to be ready to launch in some days time. There will be plenty of things to look forward to - 24/7 uptime (for real), daily & automated backups, new & improved mods, less lag :)

tl;dr: Server back shortly, same map.

Sorry again!

**Comment by GaboXandre** [comment]
Amazing news !!!
Glad to hear from you, Craig.

And Amaz, guess you won the reward:
![Alt Text](http://i.imgur.com/m9hteoj.jpg)

**Comment by RHRhino** [comment]
Yay! Good job Amaz and welcome back Craig! :smile:

**Comment by Amaz1** [comment]
No problem! I'm very glad that you are allright!

**Comment by BobHovercraft** [comment]
Awesome, glad everything's OK Craig, will look out for the server when it comes back online. :)

**Comment by nanepiwo** [comment]
@CraigyDavi Still working on making the server better than it was ever before? I would like to play on a server such as yours and continue my half-finished builds there, as others probably do too :)

**Comment by CraigyDavi** [comment]
Hi guys,

I'll start developing on the code not this Saturday - but the Saturday after then. You can expect the server to be up shortly after that!

Terribly sorry about the long wait...

**Comment by RHRhino** [comment]
Hooray! I'm looking forward visiting your server soon! :smile:

**Comment by GaboXandre** [comment]
Since owner has been found, this **bug** should be marked **fixed** :D

**Comment by CraigyDavi** [comment]
> Since owner has been found, this bug should be marked fixed :D

Right! :smile:

**Comment by Amaz1** [comment]
Yep! :D
